assets/mods/chinese-localization/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Chinese Localization",
+    "description": "Chinese Localization",
```

Comment:
Traditional or simplified?

Reply:
It's simplified.
I will replace all 'Chinese' to 'Simplified Chinese'.

Reply:
How about also adding a short usage instructions to tell players to choose "Chinese" for game version?